src/Atmos/Model/AtmosModel.jl

```diff
@@ -121,14 +123,18 @@ Where
 """
 @inline function flux_nondiffusive!(m::AtmosModel, flux::Grad, state::Vars,
                                     aux::Vars, t::Real)
-  ρinv = 1/state.ρ
+  ρ = state.ρ
+  ρinv = 1/ρ
   ρu = state.ρu
   u = ρinv * ρu
+  z  = altitude(m.orientation, aux)
 
   # advective terms
-  flux.ρ   = ρu
-  flux.ρu  = ρu .* u'
-  flux.ρe  = u * state.ρe
+  usub = subsidence_velocity(m.subsidence, z)
+  u_tot = u + usub
```

Comment:
The name `subsidence` implies that this is a vertical velocity only, and defined positive downward. So you'd want something like
```suggestion
  u_tot = u - usub * vertical_unit_vector
```
where `vertical_unit_vector` is the upward pointing vertical unit vector.

Reply:
Should I call this `subsidence_speed` then? Since the direction is already assumed?